mxcube3/ui/components/SampleView/MotorInput.js

```diff
@@ -65,6 +66,7 @@ export default class MotorInput extends React.Component {
                 step={step}
                 defaultValue={valueCropped}
                 name={motorName}
+                disabled={this.props.state === 4 || this.props.state === 3 || this.props.state <= 1}
```

Comment:
I don't understand could you elaborate on this ?

In any case we need to find a way to prevent user from typing in this input while the motor is moving, or if it is
in fault state.